tidb-lightning/tidb-lightning-faq.md

```diff
@@ -205,3 +205,49 @@ The steps are as follows:
 2. Configure the required labels for TiKV and PD.
 3. Create the placement rule policy and apply the created policy to the target table.
 4. Use TiDB Lightning to import data into the target table.
+
+## How can I use lightning and dumpling to copy a schema
+
+This is to copy both the schema definition and the table data from one schema to a new schema. For this example we will make a copy of the `test` schema into a new schema called `test2`.
+
+We first create a backup with `-B test` to only select the schema that we need.
+
+```
+tiup dumpling -B test -o /tmp/bck1
+```
+
+Then we create a `/tmp/tidb-lightning.toml` file with the following content:
+
+```
+[tidb]
+host = "127.0.0.1"
+port = 4000
+user = "root"
+
+[tikv-importer]
+backend = "local"
```

Comment:
maybe for simplicity (for example lightning's node can't access TiKV) we should recomment "tidb" backend